### PR TITLE
Functional Tests : FO - Guest checkout: Billing address when logged in (Classic & Hummingbird)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : FO - Guest checkout: Billing address when logged in (Classic & Hummingbird)
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/8876447019
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/classic/11_checkout/02_addresses/05_billingAdressWhenLoggedIn.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    it('should validate shopping cart and go to checkout page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToCheckoutPage', baseContext);
+
+      await cartPage.clickOnProceedToCheckout(page);
+
+      const isCheckoutPage = await checkoutPage.isCheckoutPage(page);
+      expect(isCheckoutPage).to.eq(true);
+    });
+
+    it('should sign in by created customer', async function () {
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/FO/classic/11_checkout/02_addresses/05_billingAdressWhenLoggedIn.ts]:
```suggestion
    it('should sign in by default customer', async function () {
```

File: tests/UI/campaigns/functional/FO/hummingbird/11_checkout/02_addresses/05_billingAdressWhenLoggedIn.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    it('should validate shopping cart and go to checkout page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToCheckoutPage', baseContext);
+
+      await cartPage.clickOnProceedToCheckout(page);
+
+      const isCheckoutPage = await checkoutPage.isCheckoutPage(page);
+      expect(isCheckoutPage).to.eq(true);
+    });
+
+    it('should sign in by created customer', async function () {
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/FO/hummingbird/11_checkout/02_addresses/05_billingAdressWhenLoggedIn.ts]:
```suggestion
    it('should sign in by default customer', async function () {
```

Comment (Progi1984) [reply tests/UI/campaigns/functional/FO/hummingbird/11_checkout/02_addresses/05_billingAdressWhenLoggedIn.ts]: Fixed

File: tests/UI/campaigns/functional/FO/classic/11_checkout/02_addresses/05_billingAdressWhenLoggedIn.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/FO/classic/11_checkout/02_addresses/05_billingAdressWhenLoggedIn.ts]: Fixed

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Pull request merged**